packages/manager/apps/catalog/src/api/GET/aapi/service.ts

```diff
@@ -0,0 +1,61 @@
+import { queryClient } from '@ovh-ux/manager-react-core-application';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import i18next from 'i18next';
+
+export const getManagerHubCatalogListQueryKey = ['/hub/catalog'];
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+// interface ResponseData {
+//   status: number;
+//   data: CatalogData;
+// }
+export type CatalogData = {
+  data: Product[];
+};
+
+export type ResponseData<T> = {
+  status: number;
+  data: T;
+  code: string;
+  // Error response
+  response?: {
+    status: number;
+    data: { message: string };
+  };
+};
+
+/**
+ * Manager-catalog endpoints : Get manager-catalog
+ */
+export const getManagerHubCatalogList = async () => {
```

Comment:
**suggestion:** No need for `Hub` prefix
```suggestion
export const getManagerCatalogList = async () => {
```

Reply:
👍